docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -160,7 +160,65 @@ For example, you could use interpolation to build the user's full name:
 }
 ```
 
-Interpolated shortcode substituted values will always be casted to strings. For example, if the user does not have a last name associated, the above full name value would be "null John".
+Interpolated shortcodes will always result to string values. For example, if the user does not have a last name associated, the above full name value would be "null John".
+
+### Coalescing
+
+Shortcode coalescing is a conditional expression of the form:
+
+```json
+"{{ <operand_1> || <operand_2> || <operand_n> }}"
+```
+
+For example, given the following template:
+
+```json
```

Comment:
🔧 Consider splitting into separate examples, as the scope and purpose of each one might be clearer. Also, not sure how JS comments render in our docs site. Github will render them as invalid lines I believe.

Reply:
Comments are rendered nicely - we already use them in the docs (see [example](https://clerk.com/docs/backend-requests/making/jwt-templates)).